src/main/java/teammates/ui/webapi/action/ActionFactory.java

```diff
@@ -116,10 +118,38 @@
         map(ResourceURIs.STUDENT_PROFILE_PICTURE, DELETE, DeleteStudentProfilePictureAction.class);
         map(ResourceURIs.INSTRUCTOR, PUT, UpdateInstructorAction.class);
         map(ResourceURIs.INSTRUCTOR, POST, CreateInstructorAction.class);
+
+        // Cron jobs; use GET request
```

Comment:
Of course, we want to disallow the body for GET.

Reply:
Not the issue either. Cron jobs using GET is a prescribed condition from GAE.